microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1489,31 +1544,46 @@
                   <p>
                       <strong>{{ _('How do we calculate your diversity value?') }}</strong>
                   </p>
-                  <p>
+                  <p class="site_specific_content site_specific_content-gut">
```

Comment:
why switch from underscores to hyphen at the end of site_specific_content-gut?  This seems likely to cause confusion ...